<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arrowproperty</class>
 <widget class="QDialog" name="arrowproperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>箭头属性</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res/res.qrc">
    <normaloff>:/img/continue.png</normaloff>:/img/continue.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>441</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string>起始组件信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_beginport">
      <property name="text">
       <string>起始端口：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_beginport"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_beginomp">
      <property name="text">
       <string>起始组件：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_begincomp"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>441</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string>终止组件信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_endport">
      <property name="text">
       <string>终止端口：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_endport"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_endcomp">
      <property name="text">
       <string>终止组件：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_endcomp"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>箭头信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_ID">
      <property name="text">
       <string>箭头ID：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit_ID"/>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>16</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Res/res.qrc"/>
 </resources>
 <connections/>
</ui>
